### PR TITLE
make faucets more accesible

### PR DESCRIPTION
**Pull request by InoMurko**
Make developer faucet more accessible to developers  

## Overview

Make developer faucet more accessible to developers  

## Changes

- Quick link
## Testing

/

### CODE REVIEW

File: boba_documentation/developer/faucets.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+description: Faucets
+---
+
+## Boba Network test token Faucet
+
+### Ethereum Rinkeby
+
+If you do not have Rinkeby ETH, you can get some from [Rinkeby Authenticated Faucet](https://www.rinkeby.io/#faucet). For Rinkeby BOBA, use the [BOBA faucet](https://gateway.testnet.avax.boba.network/wallet).
```
Comment (boyuan-chen) [inline boba_documentation/developer/faucets.md]: The URL of Rinkeby gateway is wrong.

File: boba_documentation/developer/faucets.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### Fantom Opera
+
+If you do not have Opera FTN, you can get some from [FTN Faucet](https://faucet.fantom.network/). For Opera BOBA, use the [BOBA faucet](https://gateway.testnet.bobaopera.boba.network/wallet).
+
+### BNB testnet
+
+If you do not have testnet BNB, you can get some from [BNB Faucet](https://testnet.binance.org/faucet-smart). For testnet BOBA, use the [BOBA faucet](https://gateway.testnet.bnb.boba.network/wallet).
+
+### BNB testnet
```
Comment (boyuan-chen) [inline boba_documentation/developer/faucets.md:19]: This is duplicated.

File: boba_documentation/developer/faucets.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+If you do not have Rinkeby ETH, you can get some from [Rinkeby Authenticated Faucet](https://www.rinkeby.io/#faucet). For Rinkeby BOBA, use the [BOBA faucet](https://gateway.testnet.avax.boba.network/wallet).
+
+### Avalanche Fuji
+
+If you do not have Fuji Avax, you can get some from [Fuji Faucet](https://faucet.avax.network). For Fuji BOBA, use the [BOBA faucet](https://gateway.testnet.avax.boba.network/wallet).
+
+### Fantom Opera
+
+If you do not have Opera FTN, you can get some from [FTN Faucet](https://faucet.fantom.network/). For Opera BOBA, use the [BOBA faucet](https://gateway.testnet.bobaopera.boba.network/wallet).
```
Comment (boyuan-chen) [inline boba_documentation/developer/faucets.md]: Is it Opera FTM?

Comment (InoMurko) [reply boba_documentation/developer/faucets.md]: good catch

**Review by boyuan-chen — APPROVED** [review]

**Pull request merged**